osdconfig/util.go

```diff
@@ -0,0 +1,85 @@
+package osdconfig
```

Comment:
Let's not create a util.go file. It's too generic and eventually becomes a dump ground for all miscellaneous routines and the kitchen sink.

Reply:
ok, will move those util functions around and delete util.go

Reply:
done